src/node/internal/internal_zlib.ts

```diff
@@ -14,6 +14,20 @@ function crc32(data: ArrayBufferView | string, value: number = 0): number {
   return zlibUtil.crc32(data, value);
 }
 
+
+const constPrefix = 'CONST_';
+const constants = {};
+// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+Object.entries(Object.getPrototypeOf(zlibUtil))
+  .filter(([k,]) => k.startsWith(constPrefix))
+  .forEach(([k, v]) => Object.defineProperty(constants, k.slice(constPrefix.length), {
+    value: v,
+    writable: false,
+    configurable: false,
+    enumerable: true
+  }));
```

Comment:
@anonrig, @jasnell
Here's my implementation of dynamically setting up the zlib constants. I find it kind of ugly though - let me know if you have any suggestions.

Reply:
This is ok. I personally would have just listed them out individually but this works fine also

Reply:
Fwiw, however, using `Object.defineProperties(...)` and setting multiple properties in one go there is a lot more performant than repeatedly calling `Object.defineProperty` one at a time.